saleor/graphql/page/mutations/attributes.py

```diff
@@ -197,3 +200,87 @@ def perform_mutation(cls, _root, info, **data):
             perform_reordering(page_attributes, operations)
 
         return PageTypeReorderAttributes(page_type=page_type)
+
+
+class PageReorderAttributeValues(BaseReorderAttributeValuesMutation):
+    page = graphene.Field(
+        Page, description="Page from which attribute values are reordered."
+    )
+
+    class Meta:
+        description = "Reorder page attribute values."
+        permissions = (PagePermissions.MANAGE_PAGES,)
+        error_type_class = PageError
+        error_type_field = "page_errors"
+
+    class Arguments:
+        page_id = graphene.Argument(
+            graphene.ID, required=True, description="ID of a page."
+        )
+        attribute_id = graphene.Argument(
+            graphene.ID, required=True, description="ID of an attribute."
+        )
+        moves = graphene.List(
+            ReorderInput,
+            required=True,
+            description="The list of reordering operations for given attribute values.",
+        )
+
+    @classmethod
+    def perform_mutation(cls, _root, info, **data):
```

Comment:
Maybe we can extract a lot of code from this method to `BaseReorderAttributeValuesMutation`?